modules/javafx.web/src/ios/java/com/sun/javafx/scene/web/skin/HTMLEditorSkin.java

```diff
@@ -23,7 +23,7 @@
  * questions.
  */
 
-package com.sun.javafx.scene.web.skin;
+package javafx.scene.web;
```

Comment:
The package and the file location no longer match. I think the file needs to move to `modules/javafx.web/src/ios/java/javafx/scene/web/skin/HTMLEditorSkin.java `.